### PR TITLE
Recipe for pypng

### PR DESCRIPTION
**Pull request by npanpaliya**

### CODE REVIEW

File: conda-recipes/pypng-feedstock/ci_support/local_build.sh [hunk 1]
```diff
@@ -0,0 +1,50 @@
+# (C) Copyright IBM Corp. 2018, 2019. All Rights Reserved.
```
Comment (jayfurmanek) [inline conda-recipes/pypng-feedstock/ci_support/local_build.sh]: Is this file needed?

Comment (npanpaliya) [reply conda-recipes/pypng-feedstock/ci_support/local_build.sh]: No, I mistakenly added that. Removing it.

**Comment by jayfurmanek** [comment]
https://github.com/IBM/powerai/issues/43
